src/tests/functionspace/test_cubedsphere_functionspace.cc

```diff
@@ -207,6 +207,106 @@ CASE("test copies and up/down casting") {
     }
 }
 
+CASE("Cubed sphere primal-dual equivalence") {
+
+    // Created a LFRic layout N12 cubedsphere grid.
+    // This is a global set of grid points with lonlats located at cell centres.
+    const auto grid = Grid("CS-LFR-12");
+
+    // Create domain decomposed mesh.
+    // We can generate two types of mesh, a primal and a dual.
+    // The primal mesh can have an arbitarily large halo (default = 1), whereas
+    // the dual mesh has a fixed halo size. The dual mesh is useful if you
```

Comment:
Is the comment here still valid for the dual mesh size.

Reply:
Still valid for now. I think I can make the dual mesh halo size modifiable. If the change is as simple as I think it is, I might add it to a future PR.